x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/categorization_detector/detector_cards.tsx

```diff
@@ -31,6 +31,45 @@ export const CountCard: FC<CardProps> = ({ onClick, isSelected }) => (
             id="xpack.ml.newJob.wizard.pickFieldsStep.categorizationDetectorSelect.countCard.description"
             defaultMessage="Look for anomalies in the event rate of a category."
           />
+
+          <EuiSpacer size="s" />
+
+          <FormattedMessage
+            id="xpack.ml.newJob.wizard.pickFieldsStep.categorizationDetectorSelect.countCard.description2"
+            defaultMessage="Using Count is more appropriate when categorizing all messages."
```

Comment:
Is `recommended` an overstatement? If not, then my suggestion is a bit shorter.
```suggestion
            defaultMessage="Recommended for categorizing all messages."
```